react/Header/Navigation/Navigation.js

```diff
@@ -79,7 +79,7 @@ Navigation.propTypes = {
   locale: PropTypes.string.isRequired,
   linkRenderer: PropTypes.func.isRequired,
   divider: PropTypes.bool.isRequired,
-  activeTab: PropTypes.oneOf(items.map(({ name }) => name))
+  activeTab: PropTypes.string
```

Comment:
Given the small device menu doesn't share this component, this line could be reverted right?